server/src/server.rs

```diff
@@ -334,6 +334,18 @@ impl rpc::Server for CubebServer {
     }
 }
 
+// Debugging for BMO 1594216/1612044.
+macro_rules! stream {
```

Comment:
nit: Renaming it like `try_stream` might be clear.

Reply:
Yeah, makes sense now that it returns an error instead of panicking.  Thanks, will change.